### DISCUSSION TITLE
Optimization variable problem

### DISCUSSION THREAD

**Discussion by szl-c** [lululxvi/deepxde#909] [Q&A]
Dear Lu Lu,
Thanks for the great support for DeepXDE, and it's powerful.
I want to find a location variable that uses the PDE loss function away from this location, But take a different form of the loss function around the position, like this :f(x+a)**2=f(x-a)**2+1.Do you have an idea how I should proceed ?

**Comment by szl-c** [comment]
Just explain ,I means How do you add a loss function between two points separately, except for those two points with another loss function

**Comment by lululxvi** [comment]
Can't understand your question. Could you provide more details?

**Reply by szl-c** [reply]
How do I add a loss function between two points

**Reply by lululxvi** [reply]
The two points are fixed?
